core/src/main/java/org/apache/iceberg/view/ViewVersionParser.java

```diff
@@ -42,7 +43,7 @@ static void toJson(ViewVersion version, JsonGenerator generator) throws IOExcept
 
     generator.writeNumberField(VERSION_ID, version.versionId());
     generator.writeNumberField(TIMESTAMP_MS, version.timestampMillis());
-
+    generator.writeNumberField(SCHEMA_ID, version.schemaId());
```

Comment:
nit: we should probably not remove the newline before starting to write summary part

Reply:
Fixed!